chia/types/spend_bundle.py

```diff
@@ -59,6 +60,10 @@ def fees(self) -> int:
 
         return amount_in - amount_out
 
+    @property
+    def id(self) -> SpendBundleID:
+        return bytes_to_SpendBundleID(self.get_hash())
+
     def name(self) -> bytes32:
```

Comment:
`SpendBundle.name()` has many many uses in many many files.

Reply:
In the long run, I imagine we'd want to make all those places aware of any type casting they do too